### PR TITLE
[EN/ZH] Update "Storyboarding"

### PR DESCRIPTION
**Pull request by kj415j45**
- [x] ~~en ver is a stub, keep this PR draft.~~

### CODE REVIEW

File: wiki/Storyboarding/zh.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# 制作故事板
+
+制作故事板通常需要耗费大量时间和精力，并且需要一定的动画或图片制作基础。在谱面编辑器中有一个[故事板编辑器（Storyboard Editor）](/wiki/Beatmap_Editor/Design)，在故事板编辑器中，点击`Design`栏或者使用快捷键 `F2` 都可以打开。故事板编辑器能让你使用故事板的一些基础功能，做出简单的效果。较为熟练的[故事板作者](/wiki/Storyboarding/Storyboarder)一般更倾向于使用[故事板脚本（Storyboard Scripting）](/wiki/Storyboard_Scripting) 来制作。复杂的视觉效果常常需要大量的代码才能实现，因此，许多作者会通过编程的方式直接导出故事板脚本。
```
Comment (kj415j45) [inline wiki/Storyboarding/zh.md:3]: Cherry-pick review: sync the first sentence to eng ver.
```suggestion
制作故事板通常需要耗费大量时间和精力，并且需要一定的动画或图片制作基础。在谱面编辑器中有一个[故事板编辑器](/wiki/Beatmap_Editor/Design)，在故事板编辑器中，点击`Design`栏或者使用快捷键 `F2` 都可以打开。故事板编辑器能让你使用故事板的一些基础功能，做出简单的效果。较为熟练的[故事板作者](/wiki/Storyboarding/Storyboarder)一般更倾向于使用[故事板脚本](/wiki/Storyboard_Scripting) 来制作。复杂的视觉效果常常需要大量的代码才能实现，因此，许多作者会通过编程的方式直接导出故事板脚本。
```

File: wiki/Storyboarding/Storyboarder/zh.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+tags:
+  - SBer
+---
+
+# 故事板作者
+
+故事板作者这个术语是指为[谱面](/wiki/Beatmap)制作[制作故事板](/wiki/Storyboarding)的玩家，他们使用谱面编辑器中的`Design`栏、第三方软件或[故事板脚本](/wiki/Storyboard_Scripting)来编辑[故事板](/wiki/Storyboards)。
```
Comment (kj415j45) [inline wiki/Storyboarding/Storyboarder/zh.md:8]:
```suggestion
故事板作者这个术语是指为 [谱面](/wiki/Beatmap) [制作故事板](/wiki/Storyboarding) 的玩家，他们使用谱面编辑器中的`Design`栏、第三方软件或 [故事板脚本](/wiki/Storyboard_Scripting) 来编辑 [故事板](/wiki/Storyboards)。
```

**Comment by TicClick** [comment]
@kj415j45 I've improved the stub but honestly it needs to be gone (that's a separate task though). please readjust the translation while you're at it, and I'll merge it in

File: wiki/Storyboarding/zh.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+outdated: true
+outdated_since: 0cf0d69bb66378ba1f9f000956c30bc61b92a609
+---
+
+# 制作故事板
+
+制作故事板通常需要耗费大量时间和精力，并且需要一定的动画或图片制作基础。在谱面编辑器中有一个[故事板编辑器（Storyboard Editor, SBE）](/wiki/Beatmap_Editor/Design)，点击 `Design` 栏或者使用快捷键 `F2` 都可以打开。故事板编辑器能让你使用故事板的一些基础功能，做出简单的效果如控制精灵图的移动，但做出更为复杂的效果则麻烦许多。如果想制作宽屏故事板，请在歌曲设置（`Song Setup`）对话框的设计（`Design`）标签页中勾选宽屏支持（`Widescreen Support`）。
+
+较为熟练的[故事板作者](/wiki/Storyboarding/Storyboarder)一般更倾向于使用[故事板脚本（Storyboard Scripting）](/wiki/Storyboard_Scripting) 来制作。复杂的视觉效果常常需要大量的代码才能实现，因此，许多作者会通过编程的方式直接导出故事板脚本。
```
Comment (kj415j45) [inline wiki/Storyboarding/zh.md:10]:
```suggestion
尽管内置编辑器提供了一些控制精灵图移动的方法，但更高级的效果则较为棘手，因此[故事板作者](/wiki/Storyboarding/Storyboarder)一般更倾向于使用[故事板脚本（Storyboard Scripting）和一些第三方程序](/wiki/Storyboard_Scripting)来生成故事板的代码，并提供高级的抽象功能，例如粒子系统。
```

File: wiki/Storyboarding/zh.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+---
+outdated: true
+outdated_since: 0cf0d69bb66378ba1f9f000956c30bc61b92a609
+---
+
```
Comment (kj415j45) [inline wiki/Storyboarding/zh.md:5]:
```suggestion
```

**Review by TicClick — APPROVED** [review]

**Pull request merged**